test/integration/services/common.go

```diff
@@ -10,6 +10,11 @@ import (
 	"k8s.io/kubernetes/test/e2e/framework"
 )
 
+const (
+	defaultHost            = "http://127.0.0.1:8080"
+	integrationTestAppName = "integration-test-app"
```

Comment:
thanks for fixing it. 
integrationTestAppName is unused if you can remove it.

Reply:
Yeah. I just let that variable stay since this file will be developed more and it can be omitted later.